app/models/cdl.rb

```diff
@@ -50,4 +55,24 @@ class Cdl < ActiveFedora::Base
   # including `include ::Hyrax::BasicMetadata`. All properties must
   # be declared before their values can be ordered.
   include OrderMetadataValues
+
+  private
+
+    def create_group_and_add_members
+      return if is_child
+
+      # IiifPrint::Jobs::CreateRelationshipsJob has a 10 min timeout
```

Comment:
Wondering about the timing of this?  Should we instead key off of the `CreateRelationshipsJob`?  We'll need to talk this over.

Reply:
I think we should decorate the `IiifPrint::Jobs::CreateRelationshipsJob` to conditionally launch this job.

Reply:
but the job itself has a check built in as well as retries if conditions aren't met, but I do agree with keying off of `CreateRelationshipsJob`.